shopping-cart.html

```diff
@@ -6,17 +6,99 @@
     <title>Shopping Cart Page</title>
     <link rel="stylesheet" href="./assets/CSS/all.css">
     <link rel="stylesheet" href="./assets/CSS/bootstrap.min.css">
+    <link rel="stylesheet" href="./assets/CSS/colors.css">
     <link rel="stylesheet" href="./assets/CSS/nav-footer.css">
     <link rel="stylesheet" href="./assets/CSS/shopping-cart.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
 </head>
 
 <body>
     <div data-include="navbar.html"></div>
-    <main class="h-75"></main>
-    <div data-include="footer.html"></div>
-    <script src="./assets/JS/bootstrap.min.js"></script>
-    <script src="./assets/JS/nav-footer.js"></script>
-    <script src="./assets/JS/shopping-cart.js"></script>
+    <br/>
+    <div class="container mt-4">
+        <table class="table">
+            <thead>
+                <tr>
+                    <th scope="col">Product</th>
+                    <th scope="col">Price</th>
+                    <th scope="col" class="col-md-2">Quantity</th>
+                    <th scope="col">Subtotal</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr class="align-middle">
+                    <td><img src="assets/checkout-images/TV.png" alt="LCD Monitor" width="75" /> LCD Monitor</td>
+                    <td>$650</td>
+                    <td><button type="button" class="btn border-secondary text-center">+</button>
```

Comment:
please change the order of the buttons the minus button first then the plus button